Fix query search_around_coordinates option

ajstewart opened the pull request:
- Also added test for find_sources with `search_around=True`.
- Fixes #364.

pep8speaks commented:
Hello @ajstewart! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`tests/test_query.py`](https://github.com/askap-vast/vast-tools/blob/776b9e85c600532d6a2cf960b8ac3c73c6f2e46b/tests/test_query.py):

> [Line 1440:5](https://github.com/askap-vast/vast-tools/blob/776b9e85c600532d6a2cf960b8ac3c73c6f2e46b/tests/test_query.py#L1440): [E303](https://duckduckgo.com/?q=pep8%20E303) too many blank lines (2)

ajstewart commented:
@ddobie I fixed the search around issue and also added a test of find_sources() with search around being True. This also meant updating the add_files lines in the selavy cat fixture as these were out of date.

Are you able to test it at all before merging?

ddobie commented:
I ran into an issue with the selavy meta which revealed that we actually don't use the '#' column except in the search-around option. I've gone through and reorganised things on a separate branch which can be pulled onto this one (see #366)

ajstewart commented:
Hopefully this works now!
